### DISCUSSION TITLE
Using MKL-DNN

### DISCUSSION THREAD

**Discussion by manoadamro** [PaddlePaddle/PaddleOCR#11235] [Q&A]
I have seen some comments about speeding up detection by using `enable_mkldnn`.

Eg: https://github.com/PaddlePaddle/PaddleOCR/discussions/8141

MKL-DNN has been renamed to OneDNN and even with both/either installed, I get the error:
```
AttributeError: 'paddle.fluid.libpaddle.AnalysisConfig' object has no attribute 'set_mkldnn_cache_capacity
``` 
I have tried installing it with both the intel installer and with a conda environment but no luck getting paddleocr to use it.

Is this flag `enable_mkldnn` still valid and if so, how can I get paddleOCR to use it?

**Comment by ammar-baga** [comment]
If you're using PaddleOCR on an ARM64-based device, MKL won't be available. try on x_86_64 intel or amd
